opml-importer.php

```diff
@@ -10,6 +10,9 @@
 License: GPL version 2 or later - http://www.gnu.org/licenses/old-licenses/gpl-2.0.html
 */
 
+// If a user want to import links we must enable the link manager
+add_filter( 'pre_option_link_manager_enabled', '__return_true' );
```

Comment:
This will only have effect while the importer is installed and active. As importers are - in most cases - used once and then deactivated again, I don't think this is very effective.